library/bdwgc/src/lib.rs

```diff
@@ -85,5 +85,7 @@ extern "C" {
 
     pub fn GC_finalized_total() -> u64;
 
+    pub fn GC_get_gc_no() -> u64;
```

Comment:
Is this "no" or "num"?

Reply:
It's num, but the BDWGC calls it no as shorthand :man_shrugging: